src/waypoint_cmd.cpp

```diff
@@ -111,6 +113,29 @@ Axis GetAxisForNewWaypoint(TileIndex tile)
 	}
 }
 
+/**
+ * Get the axis for a new road waypoint. This means that if it is a valid
+ * tile to build a waypoint on it returns a valid Axis, otherwise an
+ * invalid one.
+ * @param tile the tile to look at.
+ * @return the axis for the to-be-build waypoint.
+ */
+Axis GetAxisForNewRoadWaypoint(TileIndex tile)
+{
+	/* The axis for rail waypoints is easy. */
```

Comment:
Copy-paste error, but this makes more sense as:

```suggestion
	/* The axis for existing waypoints is easy. */
```